src/main/java/core/basesyntax/supplier/ColorSupplier.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax.supplier;
+
+import core.basesyntax.service.Color;
+import java.util.Random;
+
+public class ColorSupplier {
+    private Color[] colors = Color.values();
+    private Random rnd = new Random();
```

Comment:
abbreviations are worsening readability, check other vars as well
```suggestion
    private Random random = new Random();
```